mantidproject/mslice
# Replace nose with pytest

**Description of work:**
This PR replaces the usage of nose with pytest for running the Mslice tests. 

**To do**
- [x] Add instructions for running pytest from command line
- [x] Add instructions for running pytest from pycharm
- [x] Check coverage reporting still works

**To test:**
Make sure tests are run successfully, and the coverage is reported
Grep for references to nosetest


Fixes #858
